appcenter-auth/ios/AppCenterReactNativeAuth/AppCenterReactNativeAuth.m

```diff
@@ -40,4 +43,35 @@ + (void)register {
     }
 }
 
+RCT_EXPORT_METHOD(isEnabled:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    resolve([NSNumber numberWithBool:[MSAuth isEnabled]]);
+}
+
+RCT_EXPORT_METHOD(setEnabled:(BOOL)shouldEnable
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    [MSAuth setEnabled:shouldEnable];
+    resolve(nil);
+}
+
+RCT_EXPORT_METHOD(signIn:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    [MSAuth signInWithCompletionHandler:^(MSUserInformation * _Nullable userInformation, NSError * _Nullable error) {
+        if(!error) {
+            
+            /* Sign-in succeeded, convert Java result to a JavaScript result. */
+            NSMutableDictionary *dict = [[NSMutableDictionary alloc] init];
+            dict[kMSAccountId] = userInformation.accountId;
```

Comment:
What about id token and access token?

Reply:
It needs the 2.1.0 SDK or use a prerelease. Not sure if we can easily commit that (android was easy to consume prerelease).

Reply:
Using prerelease iOS sdk is doable but requires some extra steps compared to android. We should do it.